indicoio/api/prebuilt.py

```diff
@@ -60,33 +82,54 @@ def pdf_extraction(
             return job.result()
 
     def document_extraction(
-        self, data: List[str], job_results: bool = False, **document_extraction_options
+        self,
+        data: List[str] = None,
+        job_results: bool = False,
+        large_document_paths: List[str] = None,
```

Comment:
no need to separately support this. all data, if determined to be an existing file path, should be uploaded

Reply:
makes sense. handled in storage client.